config/routes.rb

```diff
@@ -4,4 +4,6 @@
       resources :skills, defaults: { format: :json }
     end
   end
+
+  get '/auth/:provider/callback', to: 'sessions#create', as: 'login'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.